typedjsonrpc/registry.py

```diff
@@ -37,11 +37,11 @@ class Registry(object):
     :type tracebacks: dict[int, werkzeug.debug.tbtools.Traceback]
     """
 
-    json_encoder = json.JSONEncoder
-    """The JSON encoder class to use.  Defaults to :class:`json.JSONEncoder`"""
+    json_encoder = json.JSONEncoder()
+    """The JSON encoder to use.  Defaults to `json.JSONEncoder`"""
```

Comment:
I think I would prefer a comment here in there tells you when this changed from a class to a object. That'll help if anyone is already using 0.1.0.